yearn/v2/vaults.py

```diff
@@ -210,4 +211,17 @@ def tvl(self, block=None):
         except PriceError:
             price = None
         tvl = total_assets * price / 10 ** self.vault.decimals(block_identifier=block) if price else None
-        return Tvl(total_assets, price, tvl)
\ No newline at end of file
+        return Tvl(total_assets, price, tvl)
+
+
+    def _needs_curve_simple():
+        curve_simple_excludes = {
+            Network.Fantom: [
+                "0xCbCaF8cB8cbeAFA927ECEE0c5C56560F83E9B7D9"
+            ]
+        }
+        needs_simple = True
+        if chain.id in curve_simple_excludes:
+            needs_simple = self.vault.address not in curve_simple_excludes[chain.id]
```

Comment:
I think this will crash if a network is not in `curve_simple_excludes`. Either include all networks in `curve_simple_excludes` or change to `curve_simple_excludes.get(chain.id, [])`